src/storage-controller/src/history.rs

```diff
@@ -255,7 +256,13 @@ mod tests {
                         ),
                         txns_shard: Default::default(),
                     },
-                    details: SourceExportDetails::None,
+                    details: SourceExportDetails::LoadGenerator(LoadGeneratorSourceExportDetails {
+                        output: LoadGeneratorOutput::Default,
+                    }),
+                    data_config: SourceExportDataConfig {
+                        encoding: Default::default(),
+                        envelope: SourceEnvelope::CdcV2,
```

Comment:
Is this correct?

Reply:
I think so? It makes the test pass and is the same envelope being used before I refactored the structure here